src/MakePotCommand.php

```diff
@@ -603,11 +604,12 @@ protected function audit_strings( $translations ) {
 				! $translation->hasExtractedComments() &&
 				preg_match( self::SPRINTF_PLACEHOLDER_REGEX, $translation->getOriginal(), $placeholders ) >= 1
 			) {
-				WP_CLI::warning( sprintf(
+				$warning = sprintf(
```

Comment:
I don't like the `WP_CLI::warning( $warning );` here, I'd prefer to rename the variable to `$message` instead.
```suggestion
				$message = sprintf(
```

Reply:
Good point, reads better. I've fixed up the original commit to avoid CS commit spam ;-)